tests/page_objects/helioviewer.ts

```diff
@@ -312,6 +328,89 @@ class Helioviewer {
         await this.page.locator('#logo').hover();
     }
 
+    /**
+    * Get the loaded date in helioviewer 
+    * @returns Date|null - Loaded date of helioviewer, it can be null if any error.
+    */
+    async GetLoadedDate(): Date|null {
```

Comment:
async functions always return type `Promise`
```suggestion
    async GetLoadedDate(): Promise<Date|null> {
```